app/domain/authentication/authn_azure/validate_application_identity.rb

```diff
@@ -30,14 +31,8 @@ def call
 
       private
 
-      def validate_xms_mirid_format
-        begin
-          required_keys = %w(subscriptions resourcegroups providers)
-          missing_keys = required_keys - xms_mirid_hash.keys
-          raise Err::ClaimInInvalidFormat, "Required keys #{missing_keys} are missing" if missing_keys.empty?
-        rescue => e
-          raise Err::ClaimInInvalidFormat, e.inspect
```

Comment:
why was this removed in refactor? Don't we want to catch any errors that are raised?

Reply:
I rescue errors in the new method for the parse. This way it's clearer which logic raised the error.